lib/taskana-cdi-example/src/test/java/pro/taskana/ExampleBootstrap.java

```diff
@@ -0,0 +1,14 @@
+package pro.taskana;
```

Comment:
why does this test class exist? Honestly I dont see any tests coming soon for our spring-boot demo